src/crs_linter/rules/ignore_case.py

```diff
@@ -0,0 +1,132 @@
+from crs_linter.lint_problem import LintProblem
+from crs_linter.rule import Rule
+
+# Define the valid operators, actions, transformations and ctl args
+OPERATORS = "beginsWith|containsWord|contains|detectSQLi|detectXSS|endsWith|eq|fuzzyHash|geoLookup|ge|gsbLookup|gt|inspectFile|ipMatch|ipMatchF|ipMatchFromFile|le|lt|noMatch|pmFromFile|pmf|pm|rbl|rsub|rx|streq|strmatch|unconditionalMatch|validateByteRange|validateDTD|validateHash|validateSchema|validateUrlEncoding|validateUtf8Encoding|verifyCC|verifyCPF|verifySSN|within".split("|")
+OPERATORSL = [o.lower() for o in OPERATORS]
+
+ACTIONS = "accuracy|allow|append|auditlog|block|capture|chain|ctl|deny|deprecatevar|drop|exec|expirevar|id|initcol|logdata|log|maturity|msg|multiMatch|noauditlog|nolog|pass|pause|phase|prepend|proxy|redirect|rev|sanitiseArg|sanitiseMatched|sanitiseMatchedBytes|sanitiseRequestHeader|sanitiseResponseHeader|setenv|setrsc|setsid|setuid|setvar|severity|skipAfter|skip|status|tag|t|ver|xmlns".split("|")
+ACTIONSL = [a.lower() for a in ACTIONS]
+
+TRANSFORMS = "base64DecodeExt|base64Decode|base64Encode|cmdLine|compressWhitespace|cssDecode|escapeSeqDecode|hexDecode|hexEncode|htmlEntityDecode|jsDecode|length|lowercase|md5|none|normalisePathWin|normalisePath|normalizePathWin|normalizePath|parityEven7bit|parityOdd7bit|parityZero7bit|removeCommentsChar|removeComments|removeNulls|removeWhitespace|replaceComments|replaceNulls|sha1|sqlHexDecode|trimLeft|trimRight|trim|uppercase|urlDecodeUni|urlDecode|urlEncode|utf8toUnicode".split("|")
+TRANSFORMSL = [t.lower() for t in TRANSFORMS]
+
+CTLS = "auditEngine|auditLogParts|debugLogLevel|forceRequestBodyVariable|hashEnforcement|hashEngine|requestBodyAccess|requestBodyLimit|requestBodyProcessor|responseBodyAccess|responseBodyLimit|ruleEngine|ruleRemoveById|ruleRemoveByMsg|ruleRemoveByTag|ruleRemoveTargetById|ruleRemoveTargetByMsg|ruleRemoveTargetByTag".split("|")
+CTLSL = [c.lower() for c in CTLS]
+
+
+class IgnoreCase(Rule):
+    """Check the ignore cases at operators, actions, transformations and ctl arguments."""
+
+    def __init__(self):
+        super().__init__()
+        self.success_message = "Ignore case check ok."
+        self.error_message = "Ignore case check found error(s)"
+        self.error_title = "Case check"
+        self.args = ("data",)
+
+    def check(self, data):
+        """check the ignore cases at operators, actions, transformations and ctl arguments"""
+        chained = False
+        current_ruleid = 0
```

Comment:
Variable `current_ruleid` is never used in this function. You should remove it or add it to message as an extra information.